backend/config.json

```diff
@@ -1,13 +1,15 @@
 {
-  "local_port": 4001,
-  "cors_origin": "*",
-  "role_distribution_logic": {
-    "mafia_role_threshold": 11,
-    "mafia_divisor_1": 6,
-    "mafia_divisor_2": 10,
-    "detective_divisor": 10,
-    "medic_divisor": 10,
-    "jester_divisor": 14
-  },
-  "night_total_vote_time_in_milliseconds": 60000
+    "local_port": 4001,
+    "cors_origin": "*",
+    "role_distribution_logic": {
+        "mafia_role_threshold": 11,
+        "mafia_divisor_1": 6,
+        "mafia_divisor_2": 10,
+        "detective_divisor": 10,
+        "medic_divisor": 10,
+        "jester_divisor": 14
+    },
+    "night_total_vote_time_in_milliseconds": 6000,
```

Comment:
is this meant to be 6 seconds?

Reply:
it was changed for integration purposes (shortend time delay)